src/interactors/interactors.test.js

```diff
@@ -39,9 +49,25 @@ describe('Interactors', () => {
   });
 
   test('unregisterDevice: should execute interactor without errors when fail to unregister thing from cloud', async () => {
-    const cloud = new Cloud();
+    const cloud = new Cloud({ removeDeviceErr: 'fail to unregister thing' });
     const unregisterDevice = new UnregisterDevice(cloud);
     await unregisterDevice.execute(mockThing);
     expect(cloudMocks.mockRemoveDevice).toHaveBeenCalled();
   });
+
+  test("updateSchema: should execute interactor correctly when update thing's schema on cloud", async () => {
+    const cloud = new Cloud();
+    const updateSchema = new UpdateSchema(cloud);
+    await updateSchema.execute(mockThing);
+    expect(cloudMocks.mockUpdateSchema).toHaveBeenCalled();
+  });
+
+  test("updateSchema: should execute interactor without errors when fail to update thing's schema on cloud", async () => {
```

Comment:
Suggestion :point_down: 

```suggestion
  test("updateSchema: should execute interactor without errors when updating a thing's schema on cloud fails", async () => {
```

Reply:
Done!